prac_05/wimbledon.py

```diff
@@ -1,30 +1,41 @@
 """
 Wimbledon
 Estimated: 1 hour
-Actual:
+Actual: 2 hours 7 minutes
 """
 
-from operator import itemgetter
-
 FILENAME = "wimbledon.csv"
 
 
 def main():
+    """Get a record of Wimbledon champions, process that into a dictionary and set then display them"""
     records = get_records_from_file(FILENAME)
-    print(records)
+    champion_to_win, country_codes = get_champion_wins_and_countries(records)
+    display_wimbledon_champions_and_countries(champion_to_win, country_codes)
+
+
+def display_wimbledon_champions_and_countries(champion_to_win, country_code):
+    """Display wimbledon champions and there wins as well as what countries have won"""
+    print("Wimbledon Champions")
+    for champion, win in champion_to_win.items():
+        print(f"{champion} {win}")
+    print(sep="")
+    print(f"These {len(country_code)} countries have won Wimbledon:")
+    print(", ".join(sorted(country_code)))
 
 
-def create_champion_wins(records):
-    """Create a dictionary with champion name paired with win(s)"""
+def get_champion_wins_and_countries(records):
+    """Create a dictionary with champion name paired with win(s) and countries code into a set"""
```

Comment:
With SRP, would it be best to split this out into two functions? My thinking behind it, creating a dictionary and the countries set is two things. Maybe worth exploring?